week1/search.py

```diff
@@ -88,13 +124,109 @@ def query():
 
 def create_query(user_query, filters, sort="_score", sortDir="desc"):
     print("Query: {} Filters: {} Sort: {}".format(user_query, filters, sort))
+    multi_match_query_obj = {
+        "multi_match": {
+            "query": user_query,
+            "fields": ["name^100", "shortDescription^50", "longDescription^20", "department^10"]
+        }
+    }
+
+    if user_query == '*':
+        multi_match_query_obj = {
+            "match_all": {}
```

Comment:
the `query_string` query allows the `*` wildcard so this wouldn't be 100% necessary
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-top-level-params